src/main/java/app/linguistai/bmvp/service/AccountService.java

```diff
@@ -158,7 +180,7 @@ public List<User> getUsers() {
         }
     }
 
-    private String encodePassword(String plainPassword) {
+    private String encodePassword(String plainPassword) { // TODO no need for this method
```

Comment:
Please remove TODO and add `@Deprecated` to the method, with a comment saying that we don't need this method. We will remove deprecated code in a future sprint